stance/main.py

```diff
@@ -59,8 +59,8 @@ def main():
 
     # Overlay the template skeletons on the user's video
     logger.info("Overlaying Video")
-    overlay.display_overlay(args.video_path, motion.template_skeletons)
-    # overlay.compare_user_skeletons(args.video_path, benchmark_zone_indices, user_skeletons, motion)
+    overlay.display_overlay(args.video_path, motion.template_skeletons, benchmark_zone_indices, user_skeletons)
+    # overlay.compare_user_skeletons(args.motion_video, benchmark_zone_indices, user_skeletons, motion)
```

Comment:
wait change this back pls

Reply:
Which line?
The comment or the method call?

Reply:
change motion_video to video_path